packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -234,6 +236,14 @@ export class RipgrepSearchInWorkspaceServer implements SearchInWorkspaceServer {
         return Promise.resolve(searchId);
     }
 
+    private getRoot(fileUri: string, rootUris: string[]): string {
```

Comment:
Can you document what this function does?

Reply:
How about using `WorkspaceService.getWorkspaceRootUri`?